### PR TITLE
Hero section styling #83

### LINKED ISSUE #80: [EP-3] Add timeline component in landing page
Implement in accordance to the Figma design. Feel free to make any enhancements if necessary, don't have to follow design too strictly.

image

Can consider MUI's timeline component: https://v4.mui.com/components/timeline/, or reference the component for events in the landing page

### LINKED ISSUE #83: [EP-3] Update Hero section styling
Implement in accordance to the Figma design. Feel free to make any enhancements if necessary, don't have to follow design too strictly. Can consider extending the exco photo to background the about section as well, just my opinion.

image

This section should fit nicely into a vertical viewport.

### PR DESCRIPTION
**Pull request by zzhikai**
Implemented

- Insert Exco Picture behind Title and About Us portion
- Added Logo to About Us portion
- Edited Statistic Section
- Added maxWidth around details portion(Title/About us) 
Screenshot 2021-10-30 at 3 17 24 AM

Implemented

- History component( reference from Timeline )
- Mock history data

To be Implemented

- Image Styling

image

#83 #80

### CODE REVIEW

File: components/History/History.module.scss [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  }
+
+  .timelinedate,
+  .timelinedescription {
+    font-size: $font-size-1;
+  }
+
+  .icon {
+    background-color: #6b95df;
+    //#a8acae;
```
Comment (jerrenss) [inline components/History/History.module.scss:55]: Let's remove unused comment

File: styles/pages/Home.module.scss [hunk 2]
```diff
@@ -84,6 +85,9 @@
     margin-left: 10rem;
 
     .details {
+      background: linear-gradient(rgba(255, 255, 255, 0.6), rgb(255, 255, 255)),
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:88]: Can we extend the image to cover the entire width of the screen? Currently it is only in the center.

image

File: styles/pages/Home.module.scss [hunk 3]
```diff
@@ -124,15 +128,25 @@
     }
   }
 
+  .aboutUsContainer {
+    display: flex;
+    flex-direction: row;
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:133]: flex-direction is default row, so we don't need this line

File: styles/pages/Home.module.scss [hunk 4]
```diff
@@ -124,15 +128,25 @@
...
+  }
+
   .aboutUs {
     display: flex;
+    flex-direction: row;
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:138]: same as above

File: styles/pages/Home.module.scss [hunk 5]
```diff
@@ -419,37 +433,44 @@
     flex-direction: column;
     padding: 48px 16px 48px 16px;
     border-radius: 8px;
+    align-items: center;
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:436]: I don't think we need lines 436, 437 and 440. Dosen't affect the styling hmm

File: styles/pages/Home.module.scss [hunk 6]
```diff
@@ -419,37 +433,44 @@
...
+
     color: $primary-black-1;
-    background-color: #f5f5f5;
+    background-color: #ffffff;
 
     .statRow {
-      width: 800px;
+      width: 1200px;
+      display: flex;
+      flex-direction: row;
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:445]: flex-direction is default row, so we don't need this line

File: styles/pages/Home.module.scss [hunk 7]
```diff
@@ -419,37 +433,44 @@
...
       }
       .statText {
         text-align: center;
-        padding-left: 15%;
+        //padding-left: 15%;
         width: auto;
       }
     }
-    .statColRight {
+    /*.statColRight {
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:467]: Can remove this commented section if not used

File: styles/pages/Home.module.scss [hunk 8]
```diff
@@ -419,37 +433,44 @@
...
 
     .statRow {
-      width: 800px;
+      width: 1200px;
+      display: flex;
+      flex-direction: row;
+
     }
 
     .statColLeft {
```
Comment (jerrenss) [inline styles/pages/Home.module.scss:449]: Can rename this class, since there's no left and right anymore

**Review by jerrenss — CHANGES_REQUESTED** [review]
Can also help to check that the site looks good on mobile? You can use the Chrome Dev Tools to take a look at different viewport sizes. 

Hero section seems abit crammed here
image

There's a white gap on the right, probably an element overflowed the parent width. Might need to check the cause
image

Other than that, good work and effort!

File: components/History/History.module.scss (hunk 1 above)
Comment (zzhikai) [reply components/History/History.module.scss]: Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1

File: styles/pages/Home.module.scss (hunk 3 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss]: Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1

File: styles/pages/Home.module.scss (hunk 5 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss]: Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1

File: styles/pages/Home.module.scss (hunk 6 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss]: Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1

File: styles/pages/Home.module.scss (hunk 7 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss]: Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1

File: styles/pages/Home.module.scss (hunk 4 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss]: Resolved in commit 607f5a99ec73425562b5fd8c58e46dd6586477d1

File: styles/pages/Home.module.scss (hunk 8 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss]: Resolved in commit 2262237a957aa871503cb28f631e659323cbee28

File: styles/pages/Home.module.scss (hunk 2 above)
Comment (zzhikai) [reply styles/pages/Home.module.scss:87]: Fixed White gap bug in commit 6dbaa04758abfb94ed768e2515a709435756282e
Fixed image in commmit f3a42128f5e13b676233930f5813fda992f59498

**Comment by jerrenss** [comment]
Closed, as we are transiting to new design

**Pull request closed without merging**